arwen/cryptoapi/cryptoei.go

```diff
@@ -344,6 +374,7 @@ func v1_4_verifyCustomSecp256k1(
 	runtime := arwen.GetRuntimeContext(context)
 	crypto := arwen.GetCryptoContext(context)
 	metering := arwen.GetMeteringContext(context)
+	runtime.SetInitialGasInGasTrace(verifyCustomSecp256k1Name)
```

Comment:
please do not do this. There are only 2 UseGas in this function and you can simply use runtime.TraceGas(gasToUse)